### PR TITLE
Add Info sensors for Smlight integration

### PR DESCRIPTION
**Pull request by tl-sl**
## Proposed change

Add info sensors to SMLIGHT integration. These include the following:
- IP Address
- Coordinator mode
- Firmware channel - for updates
- Zigbee type/mode

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/125482
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Enhanced documentation for the SLZB-06 adapter, now including detailed descriptions of new sensors: Connection mode, Firmware channel, and Zigbee type.
- **Documentation**
	- Reformatted sensor descriptions for improved clarity and consistency.

### CODE REVIEW

File: source/_integrations/smlight.markdown [hunk 1]
```diff
@@ -40,14 +40,17 @@ You need a supported SLZB-06 adapter.
...
-- **Zigbee Uptime** - Uptime of Zigbee connection to ZHA/Z2M
-- **RAM Usage** - Monitor RAM Usage
-- **FS Usage** - Monitor filesystem usage
+- **Core temperature** - Temperature of core ESP32
+- **Zigbee temperature** - Temperature of Zigbee CC2652 or EFR32 chip
+- **Core uptime** - Uptime of Core device
+- **Zigbee uptime** - Uptime of Zigbee connection to ZHA/Z2M
+- **RAM usage** - Monitor RAM Usage
+- **FS usage** - Monitor filesystem usage
+- **Connection mode** -  Connection mode - Ethernet, Wi-Fi or USB
```
Comment (c0ffeeca7) [inline source/_integrations/smlight.markdown]:
```suggestion
- **Connection mode** -  Connection mode - Ethernet, Wi-Fi, or USB
```

File: source/_integrations/smlight.markdown [hunk 2]
```diff
@@ -40,14 +40,17 @@ You need a supported SLZB-06 adapter.
...
 - **Ethernet** - Ethernet connection status
 - **Wi-Fi** - Wi-Fi connection status
+- **Firmware channel** - Channel for updates, stable or development firmware or currently installed firmware.
```
Comment (c0ffeeca7) [inline source/_integrations/smlight.markdown]:
```suggestion
- **Firmware channel** - Channel for updates, stable, or development firmware or currently installed firmware.
```

File: source/_integrations/smlight.markdown [hunk 3]
```diff
@@ -40,14 +40,17 @@ You need a supported SLZB-06 adapter.
...
+- **Zigbee type** - Current mode of Zigbee chip. Coordinator, router or Thread. Only works with official firmware installed via OTA in SMLIGHT web UI.
```
Comment (c0ffeeca7) [inline source/_integrations/smlight.markdown]:
```suggestion
- **Zigbee type** - Current mode of Zigbee chip. Coordinator, router, or Thread. Only works with official firmware installed via OTA in SMLIGHT web UI.
```

File: source/_integrations/smlight.markdown [hunk 4]
```diff
@@ -40,14 +40,17 @@ You need a supported SLZB-06 adapter.
...
-- **Zigbee Uptime** - Uptime of Zigbee connection to ZHA/Z2M
-- **RAM Usage** - Monitor RAM Usage
-- **FS Usage** - Monitor filesystem usage
+- **Core temperature** - Temperature of core ESP32
+- **Zigbee temperature** - Temperature of Zigbee CC2652 or EFR32 chip
+- **Core uptime** - Uptime of Core device
+- **Zigbee uptime** - Uptime of Zigbee connection to ZHA/Z2M
+- **RAM usage** - Monitor RAM Usage
+- **FS usage** - Monitor filesystem usage
+- **Connection mode** -  Connection mode - Ethernet, Wi-Fi, or USB
```
Comment (c0ffeeca7) [inline source/_integrations/smlight.markdown:49]: what does this connection mode do?

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @tl-sl :+1:

File: source/_integrations/smlight.markdown (hunk 4 above)
Comment (darkxst) [reply source/_integrations/smlight.markdown:49]: This how the SLZB-06x device is connected. USB mode it acts like a USB Zigbee dongle, but the web interface (and  HA connection) falls back to ethernet or wifi if available.

**Comment by tl-sl** [comment]
Thanks @c0ffeeca7

**Pull request merged**